platform/mikro-e/Readme.md

```diff
@@ -3,16 +3,11 @@
 ## Overview
 
 This section explains how to build Contiki code for Mikro-E Clicker and to flash the compiled application image on Mikro-E Clicker.
-
-## Revision History
-
-| revision  | changes from previous revision |
-|---------- |------------------------------- |
-| 1.0.0     | Initial release                |
+Please refer <a href="https://docs.creatordev.io/clicker/guides/quick-start-guide/">6LoWPAN Clicker Quick Start Guide</a>
```

Comment:
I noticed it was done in this way a bit down in the same file. Could you change to markdown syntax as well ?